zypp/zypp/ContentType.h

```diff
@@ -0,0 +1 @@
+#include <zypp-core/ContentType.h>
```

Comment:
Should we add a deprecation message? Either by a comment, some Doxygen deprecation tag, or `#warning This file is deprecated, include <zypp-core/ContentType.h> instead"`?

Reply:
No, the include files are there to stay for libzypp. We will just not have them in the next-gen libzypp